migrations/v2.js

```diff
@@ -0,0 +1,131 @@
+import {
+  describe,
+  whereFromPlugin,
+  whereContent,
+  mutateContent,
+  checkContent,
+  updatePlugin,
+  getCourse,
+  getComponents
+} from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('adapt-contrib-textInput - v2.0.0 to v2.0.2', async () => {
+  let components;
+  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.0.2', { name: 'adapt-contrib-textInput', version: '<2.0.2' });
```

Comment:
A recent change just to stop it migrating anything below v2. 
```suggestion
  whereFromPlugin('adapt-contrib-textInput - from v2.0.0 to v2.0.2', { name: 'adapt-contrib-textInput', version: '>=2.0.0 <2.0.2' });
```

Reply:
This would need to be specified on all tasks rather than just the first, so ignoring for now as agreed offline.